pitch.md

```diff
@@ -23,38 +79,101 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+For now, and possibly always, I want to focus this language on beginners.
+With that in mind, I think the user will only be able to describe a few higher
+level LED actions like cascade, wave, fade, flash, and strip color. Various
+commands will require different properties but some would be color of the
+action (or range of colors for fade etc.), for how long the action should last,
+the brightness of the LEDs, and how wide the action should be - a 6 led wide 
+cascade or a single led etc.
+
+The value of the properties vary but time will be a length in seconds or 
+minutes or both. Color will either be a single RGB/HSV/HSL color or a range of 
+them and width will be an integer. 
+
+Additionally, there will be a way connect one composition to another. Similar
+to a `Jump` in Assembly, this will be the main way of connecting smaller
+compositions together into a more complex one. 
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+This seems to be similar to a Data interpretation computational model. 
+Theoretically when a program runs the first task will be to build the code for
+each block of code. Then it will piece this code together based on what the user
+states to be run. This would compile the language into Arduino machine code for
+use on an Arduino to control the LEDs. There could definitely be a lot of
+Arduino errors like issues connecting to the device, sending incorrect color 
+values to the leds, or C errors like SEGFAULT, which would all probably be 
+communicated to the user through a console of sorts.
```

Comment:
I feel if your able to check for errors in your DSL, it should be easy to avoid segfaults and other stuff. I think for a DSL that is wrapping C code, one benefit would be for the user to not go through the headaches of C.

This might be fairly hard to do, so we'll see as the project develops.